package.json

```diff
@@ -7,9 +7,17 @@
   "directories": {
     "test": "test"
   },
+  "files": [
+    "dist/bundle.js",
+    "API.md",
+    "LICENSE",
+    "README.md",
+    "lib/",
+    "types.d.ts"
+  ],
   "scripts": {
     "test": "npm run test:browser:cleanup && npm run test:lib && npm run test:parseFromUrl && npm run cover:report && npm run test:browser",
-    "bundle": "browserify \"lib/browser.js\" | uglifyjs > \"dist/bundle.js\"",
+    "bundle": "npx mkdirp \"dist\" && browserify \"lib/browser.js\" | uglifyjs > \"dist/bundle.js\"",
```

Comment:
Since some npm version it is installed together with npm, but I don't know since which one, probably since version 6. All in all it would be better if you added `mkdirp` to dev dependencies.

+ question: does `mkdirp` throw an error when a folder exists or does it reset the content/ skip creation?

Reply:
it works like `mkdir -p` so basically no error if dir exists

so what, I'm adding npx to devDeps, right?

Reply:
`mkdirp` is node package, so that should be inside `devDependencies`, no `npx`.

Reply:
I mean, it cound be `npx` 😛 
but yeah, better just add mkdirp directly 🤦🏼